tyk-headless/templates/deployment-gw-repset.yaml

```diff
@@ -109,7 +109,7 @@ spec:
         workingDir: /opt/tyk-gateway
         ports:
         - containerPort: {{ .Values.gateway.containerPort }}
-        {{- if .Values.gateway.control.enabled -}}
+        {{- if .Values.gateway.control.enabled }}
```

Comment:
@gwotg By enabling the control in the gateway, the containerPort number does not change to 9696 and still runs on 8080. It's suppose to be structured this way:

`      {{- if .Values.gateway.control.enabled }}
        - containerPort: {{ .Values.gateway.control.containerPort }}
        {{- else }}
        - containerPort: {{ .Values.gateway.containerPort }}
        {{- end }}`